storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -139,15 +143,8 @@ static bool reopen_log_files() {
   log.m_log_flags = log_flags;
   log.m_log_uuid = log_uuid;
   log.m_files = std::move(files);
-  auto file = log.m_files.find(log_get_checkpoint_lsn(log));
-  if (file == log.m_files.end()) {
-    xb::error() << "Cannot find file with LSN " << log_get_checkpoint_lsn(log)
-                << ". Required file has been deleted by server.";
-    xb::error() << "Please consider increasing your innodb_redo_log_capacity "
-                   "or using redo log archiving.";
-    return (false);
-  }
-  log_encryption_read(log);
+  auto file = log.m_files.front();
+  log_encryption_read(log, file);
```

Comment:
Better to check return value from encryption read? is it guaranteed that we always read encryption info?

Reply:
yes we are doing that in the latest commit.